app/services/katello/pulp3/repository/yum.rb

```diff
@@ -20,6 +20,12 @@ def distribution_options(path)
           }
         end
 
+        def mirror_remote_options
+          {
+            policy: smart_proxy.download_policy
```

Comment:
this doesn't handle the 'inherit' case, see:https://github.com/Katello/katello/blob/master/app/services/katello/pulp/repository/yum.rb#L69-L76